plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -122,12 +122,6 @@ class Conan(
     private val config: ConanConfig
 ) : PackageManager("Conan") {
     companion object {
-        internal val DUMMY_COMPILER_SETTINGS = arrayOf(
```

Comment:
I think it's fine (and even less confusing) to squash this to the previous commit, saying to replace hard-coded dummy compiler settings with proper ones. Otherwise there's an intermediate commit with both dummy and detected settings, which does not make sense.